compiler/src/codegen/compcore.re

```diff
@@ -3311,8 +3267,7 @@ let compile_main = (wasm_mod, env, prog) => {
   if (Grain_utils.Config.use_start_section^) {
     Function.set_start(wasm_mod, start);
   } else {
-    ignore @@
-    Export.add_function_export(wasm_mod, grain_start, Comp_utils.grain_start);
+    ignore @@ Export.add_function_export(wasm_mod, start_name, grain_start);
```

Comment:
Should we be doing a check here and warn and override if a user `_start` exists?

Reply:
Maybe, but that's not for this PR.